Misc/NEWS.d/next/Library/2024-05-02-04-27-12.gh-issue-118500.pBGGtQ.rst

```diff
@@ -0,0 +1 @@
+Add :mod:`pdb` support for zipapps
```

Comment:
should there be a documentation feature for this feature? A whatsnew entry?

Reply:
I don't think there's a proper place to document this in the pdb docs, as this should be the intuitive way to work with pdb. If you can do `python app.pyz`, you should be able to do `python -m pdb app.pyz`, just like a normal python script. I do agree that there should be a whatsnew entry for it and I'll add it.